### DISCUSSION TITLE
How's the performance  of `createServerFn` in Tanstack Start?

### DISCUSSION THREAD

**Discussion by TheOrdinaryWow** [TanStack/router#3066] [General]
I wonder how's the performance of `createServerFn` compared to web frameworks like hono or so.

**Comment by schiller-manuel** [comment]
what kind of performance?

**Reply by TheOrdinaryWow** [reply]
> what kind of performance?

eg. requests per second?

**Reply by aarjithn** [reply]
> > what kind of performance?
> 
> eg. requests per second?

depends on what you are doing in the fn.. ?
